### ISSUE TITLE
accessibility: missing textual description for buttons

### ISSUE THREAD

**Issue by mfranzke** [calcom/cal.com#17185]
### Issue Summary

The buttons to switch to the previous or next month in the BookEventForm don't include a (at least visually hidden) textual description, that could be e.g. used for by screenreader, compare to the following screenshot:
<img width="167" alt="image" src="https://github.com/user-attachments/assets/deb812f9-baac-4b52-b7af-1dcfa2e16d35">

### Steps to Reproduce

1. Go to a book event form.

### Actual Results

You'll recognize e.g. by opening the _Accessibility_ sub-tab underneath the _Elements_ section within Google Developer tools that the `button` element doesn't include any textual description, compare to the following screenshot:
<img width="1285" alt="image" src="https://github.com/user-attachments/assets/1dd1a753-9f2f-4a49-82a9-ec62124bfdac">

### Expected Results

These buttons do include (at least a visually hidden) description.

### Technical details

Not environment related.

### Evidence

Go to _Accessibility_ sub-tab underneath the _Elements_ section within Google Developer tools after inspecting the `button` element and check for a textual description:
<img width="1285" alt="image" src="https://github.com/user-attachments/assets/1dd1a753-9f2f-4a49-82a9-ec62124bfdac">

**Comment by bitbrief** [comment]
can i work on this ?
\assign

**Comment by shivambhatia604** [comment]
As per the lighthouse report buttons do not have an accessible name.
<img width="1172" alt="image" src="https://github.com/user-attachments/assets/3c168369-ec41-4e91-9675-4f62d363194b">

Solution: Add aria-label attribute
<img width="1288" alt="Screenshot 2024-10-20 at 1 36 17 PM" src="https://github.com/user-attachments/assets/e50f1097-59d8-4e78-99eb-2a6bb9025907">

**Comment by mfranzke** [comment]
@shivambhatia604 and @anikdhabal thanks a lot for providing and approving a fix to this issue. If I'm not mistaken by how translation for labels works regarding the cal.com-software, it's mainly using the `{t("…")}`-method like e.g. in lines 124 and 126 of the same file, compare to https://github.com/calcom/cal.com/pull/17198/files#diff-ab1645a11e7d023d1d996506d00a37da0ad35902e71404ba693baa995380bbbfR124-R126

So in case that we're providing static, english labels within the source code by https://github.com/calcom/cal.com/pull/17198, we're providing the first half of the fix, and still would need to make it localizable.

**Comment by mfranzke** [comment]
> @shivambhatia604 and @anikdhabal thanks a lot for providing and approving a fix to this issue. If I'm not mistaken by how translation for labels works regarding the cal.com-software, it's mainly using the `{t("…")}`-method like e.g. in lines 124 and 126 of the same file, compare to https://github.com/calcom/cal.com/pull/17198/files#diff-ab1645a11e7d023d1d996506d00a37da0ad35902e71404ba693baa995380bbbfR124-R126
> 
> So in case that we're providing static, english labels within the source code by #17198, we're providing the first half of the fix, and still would need to make it localizable.

I've opened https://github.com/calcom/cal.com/pull/17207 for this.
